dev/scripts/src/alluxio.org/build-distribution/cmd/generate-tarball.go

```diff
@@ -240,6 +240,13 @@ func addAdditionalFiles(srcPath, dstPath string, hadoopVersion version, version
 		run(fmt.Sprintf("adding %v", path), "cp", path, filepath.Join(dstPath, path))
 	}
 
+	if !fuse {
```

Comment:
i'm not familiar with building fuse, just a sanity check if the client jar should be put into place for fuse

Reply:
no client jar for fuse. you can see that the client jar file itself will not be in the `pathsToCopy` list